eng/scripts/run_tests.ps1

```diff
@@ -0,0 +1,80 @@
+Param(
+    [string] $serviceDir
+)
+
+$cwd = Get-Location
+
+# 0. Find all test directories
+Write-Host "Finding test directories..."
+$testDirs = . $PSScriptRoot/get_test_dirs.ps1 -serviceDir sdk/$serviceDir
```

Comment:
If you mean to execute the script, then you shouldn't also dot source it.

```suggestion
$testDirs = $PSScriptRoot/get_test_dirs.ps1 -serviceDir sdk/$serviceDir
```

Reply:
Doing this throws an error: `You must provide a value expression following the '/' operator.
```
ParserError: C:\Users\Sean Kane\go\src\github.com\Azure\azure-sdk-for-go\eng\scripts\run_tests.ps1:9
Line |
   9 |  $testDirs = $PSScriptRoot/get_test_dirs.ps1 -serviceDir sdk/$serviceD …
     |                            ~
     | You must provide a value expression following the '/' operator.
```